src/test/qa/dataset.js

```diff
@@ -623,7 +623,6 @@ async function set_attribute_randomizer() {
     // putObjectTagging - 50%
     // copyObject - 50%
     let useCopy = Math.floor(Math.random() * 2) === 0;
-    useCopy = true; //currently doing only copy due to bug #3228
```

Comment:
Does it actually works? Or do we sent NotImplemented.
Verify manually, if works we can remove the true, if not it should stay